src/components/banner/banner.tsx

```diff
@@ -1,19 +1,68 @@
 'use client';
 
-import { Box, Typography } from '@mui/material';
+import { Box, Button, Container, Typography } from '@mui/material';
+import { Abhaya_Libre, Lato } from 'next/font/google';
+import Image from 'next/image';
 import React from 'react';
 import './banner.scss';
 
+const abhayaLibre = Abhaya_Libre({
```

Comment:
These should be in scss files.

Reply:
These were seemingly removed in https://github.com/OpenBCca/openbc-web/pull/92/commits/297792e8aa6e108354a56e0cebfe14e20f12f1bf#diff-0ea735de30cd34bc5490cb86423f206d8ebba749dc576edb47d550d4d5a45e6eL9-L17, and Lato defined at the higher `app/layout.tsx` file https://github.com/OpenBCca/openbc-web/pull/92/commits/297792e8aa6e108354a56e0cebfe14e20f12f1bf#diff-29d0a55f5ca1a9951042587f8b891bb4cd8b204f5bc855dc2468ed44cde2f5b3R8-R11